tracing-core/src/field.rs

```diff
@@ -495,6 +495,12 @@ impl<T: fmt::Display> fmt::Debug for DisplayValue<T> {
     }
 }
 
+impl<T: fmt::Display> fmt::Display for DisplayValue<T> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.0)
```

Comment:
Take it or leave it: consider
```suggestion
        fmt::Display::fmt(f, self.0)
```
instead --- there's some small additional overhead to using a format string in `write!` vs just calling the `Display` implementation directly.

Reply:
I take it. Thanks for the suggestion.